resources/js/store/adminStore.js

```diff
@@ -131,6 +131,15 @@ export const useAdminStore = defineStore('admin', {
             this.bannedUsers = data.banned_users;
         },
 
+        /**
+         * Closes the UserReport (trashes it in the baclend)
+         * @param {hurrdurr add type}
```

Comment:
hurrdurr add the type please. It's a `ReportedUser`
And fix the typo in baclend